config.default.yaml

```diff
@@ -108,6 +108,22 @@ electricity:
 
   powerplants_filter:
 
+    # TODO doublecheck energy carriers in PyPSA-Earth
+  estimate_renewable_capacities:
+    # Renewable capacities are based on existing capacities reported by IRENA
+
+    # Reference year, any of 2000 to 2020
+    year: 2020
+    # Artificially limit maximum capacities to factor * (IRENA capacities),
+    # i.e. 110% of <years>'s capacities => expansion_limit: 1.1
+    # false: Use estimated renewable potentials determine by the workflow
+    expansion_limit: false
```

Comment:
@energyLS as I see one application of IRENA stats is to use the stats to define the **expansion** (defines p_nom_max + p_nom_min). This is currently named expansion_limit = true. Since both p_nom_min & max are defined maybe we should rename it to `set_renewable_expansion()`?

The other application of the IRENA stats is to scale the minimum expansion (p_nom_min) of a technology. For instance, to replicate what assets are already installed in today's network as a starting point for the optimization. This happens if I see correctly by default when the config defines `expansion_limit: false`. Now we are missing the option of pure greenfield optimization.

Any idea how we could make these three cases a bit clearer also in the config:
- (set p_nom_min + p_nom_max) * scale factor
- (set p_nom_min) * scale factor
- greenfield (no p_nom_min and no p_nom_max)

Reply:
Hey @pz-max I totally agree, we need to make this clearer esp. in the config.

I see the current options slightly different, or maybe I just put it in another syntax:
- Setting `expansion_limit=false` leads to: `p_nom_min = IRENA_capacities`
- Setting `expansion_limit=scale_factor` additionally leads to: `p_nom_max = p_nom_min * scale factor`
- Greenfield is possible when not calling the function

I do not fully understand what is meant by the expansion `(set p_nom_min + p_nom_max) * scale factor`, it currently does not make sense to me

Reply:
```python  

estimate_renewable_capacities:
    stats: "irena"  # False, = greenfield expansion, 'irena' uses IRENA stats to add expansion limits
    year: 2020  # Available years for IRENA stats 2000 to 2020
    p_nom_min: 1  # any float, scales the minimum expansion from stats
    p_nom_max: false,  # sets the expansion constraint, false, deactivate this option, float scales the p_nom_min factor accordingly 

```